boost-gradle/src/test/groovy/BoostPackageJaxRS20Test.groovy

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.junit.BeforeClass
+import org.junit.AfterClass
+import org.junit.Test
+
+import java.io.File
+import java.io.IOException
+import java.io.BufferedReader
+import java.io.FileReader
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertTrue
+
+import static org.gradle.testkit.runner.TaskOutcome.*
+
+import org.apache.commons.httpclient.HttpClient
+import org.apache.commons.httpclient.HttpStatus
+import org.apache.commons.httpclient.methods.GetMethod
+
+public class BoostPackageJaxRS20Test extends AbstractBoostTest {
+
+    static File resourceDir = new File("build/resources/test/jaxrsTestApp")
+    static File testProjectDir = new File(integTestDir, "BoostPackageJaxRS20Test")
+    static String buildFilename = "testJaxrs20.gradle"
+
+    private static String URL = "http://localhost:9080/api/hello"
+
+    private static final String JAXRS_20_FEATURE = "<feature>jaxrs-2.0</feature>"
+    private static String SERVER_XML = "build/wlp/usr/servers/BoostServer/server.xml"
+
+    @BeforeClass
+    public static void setup() {
+        createDir(testProjectDir)
+        createTestProject(testProjectDir, resourceDir, buildFilename)
+    }
+
+    @AfterClass
+    public static void teardown() {
+    
+        BuildResult result = GradleRunner.create()
+            .withProjectDir(testProjectDir)
+            .forwardOutput()
+            .withArguments("boostStop", "-i", "-s")
+            .build()
+       
+        assertEquals(SUCCESS, result.task(":boostStop").getOutcome())
+    }
+
+    @Test
```

Comment:
Are we guaranteed this test runs first or do we need to force somehow?

Reply:
Should be in same order as they are in this test file.

Reply:
Is that enforced by Gradle / Groovy specifically?  I know when using plain-old Java in Liberty FAT, the IBM JDK would use declaration order whereas the Oracle JDK would use alpha order.

Reply:
I'll move the build part into the setup method. Shouldn't matter once it's there.